src/index.html

```diff
@@ -11,7 +11,187 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="form-fieldset-first">
+        <legend>Personal information</legend>  
+        <div class="form-field">  
+          <label for="Surname">
+            Surname:
+          </label>
+          <input
+            name="Surname"
+            type="text"
+            id="Surname"
+            autocomplete="off"
+          >
+        </div>
+        <div class="form-field">   
+          <label for="Name">
+            Name:
+          </label>
+          <input
+            name="Name"
+            type="text"
+            id="Name"
+            autocomplete="off"
+          >
+        </div>
+        <div class="form-field">
+          <label for="Age">
+            How old are You?
+          </label>
+          <input
+            name="Age"
+            type="number"
+            id="Age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </div>
+        <div class="form-field">
+          <label for="DateOfBirth">
+            Full date of birth:
+          </label>
+          <input
+            name="DateOfBirth"
+            type="date" 
+            id="DateOfBirth"
+          >
+        </div>
+        <div class="form-field-last-in-fieldset">
```

Comment:
```suggestion
        <div class="form-field">
```
use `:last-child` pseudoselector